quota.go

```diff
@@ -0,0 +1,39 @@
+package goisilon
+
+import (
+
+	papi "github.com/emccode/goisilon/api/v1"
+)
+
+type Quota *papi.IsiQuota
+
+// GetQuota returns a specific quota by path
+func (c *Client) GetQuota(id, name string) (Quota, error) {
```

Comment:
This was originally to conform to the style that rex-ray used.  However, I stopped using that with all the other goisilon functions, but missed this one.  Fixed it to be consistent with the other functions in goisilon.